.github/workflows/ci.yml

```diff
@@ -47,7 +47,11 @@ jobs:
     - name: Checkout code
       uses: actions/checkout@v2
     - name: Set up the environment
-      run: cp .env.example .env
+      run: |
+        cp .env.example .env
+        pip3 install podman-compose
+    - name: Build using podman-compose
+      run: podman-compose build
     - name: Build using docker-compose
       run: docker-compose build
```

Comment:
```suggestion
```

Reply:
What's the difference? :eyes:

Reply:
I thought the purpose of this PR is to replace the docker-compose with podman-compose, the suggestion is to remove the `docker-compose` part :)

Reply:
if you want to ensure it works both ways - then ignore me

Reply:
Yeah, @marleystipich2 is using docker still, so we're making sure both developer setups work.